docs/Developers/Git_Basics.rst

```diff
@@ -402,7 +402,7 @@ Common Git Issues
 -----------------
 
 Sometimes, you use ``git add -A`` and add files you didn't want to your push
-(often after rebasing) and push it to the remote. Here ,is a short outline of,
+(often after rebasing) and push it to the remote. Here is a short outline of,
```

Comment:
shouldn't the comma after `short outline of` be removed? seems grammatically incorrect